src/plugins/geomuxp/index.js

```diff
@@ -144,10 +144,18 @@ geomux.prototype.start = function start()
         }
     }
 
-    // Create all launch options
-    var launch_options = [
+    var launch_options = [];
+    if( process.env.USE_MOCK == 'true' ){
```

Comment:
Just curious, could this be squashed down into:
```javascript
if( process.env.USE_MOCK != 'true' ) {
  launch_options = launch_options.concat([
    'nice',
    '--19'
  ])
}

launch _options = launch_options.concat([...
```
Basically to get rid of that empty else?